app/pages/patient/patientinfo.js

```diff
@@ -298,9 +297,10 @@ var PatientInfo = React.createClass({
     } else {
       return 'Birthdate not known';
```

Comment:
Oh yeah, this is exactly the kind of method behavior that makes me not a huge fan of moment. :( The oddity is definitely acceptable for now, and in the future we can either adjust sundial in that way, or maybe not use moment's method for this, especially if this is the only place we're consuming that sundial method, which may be the case.